includes/config/config.plugin.php

```diff
@@ -84,4 +84,20 @@
 
 	// The time, in seconds, that log files are kept for.
 	'max_log_age'          => 30 * DAY_IN_SECONDS,
+
+	/**
+	 * Plugin_notices is used to add "unread" notice counts to various
+	 * UI locations within boldgrid plugin. This config is used by
+	 * Boldgrid\Library\Library\NoticeCounts
+	 */
+	'pages'                => [
+		'boldgrid-backup-premium-features',
+	],
+	'page-notices'         => [
```

Comment:
If you look at the keys higher up in the config, they use `_` instead of `-`.

We can change `page-notices` to `page_notices`.